extensions/amp-story/1.0/test/test-amp-story.js

```diff
@@ -367,8 +370,139 @@ describes.realWin('amp-story', {
           return expect(replaceStub).to.not.have.been.called;
         });
   });
-  describe('amp-story continue anyway', () => {
 
+  it('should not set first page to active when rendering paused story', () => {
```

Comment:
[per comment above] is it this case?

Reply:
It is, but it tests a different piece of code.
This tests the case where the story is paused before being rendered. It makes sure we don't play the media on the first page, which would happen in the ``switchTo`` method of ``amp-story.js``.